### PR TITLE
GLT-2812: added interface for editing instrument models

### PR DESCRIPTION
**Pull request by djcooke**

### CODE REVIEW

File: miso-web/src/main/webapp/WEB-INF/pages/editInstrumentModel.jsp [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<%--
+  ~ Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+
+  ~ MISO project contacts: Robert Davey @ TGAC
```
Comment (avarsava) [inline miso-web/src/main/webapp/WEB-INF/pages/editInstrumentModel.jsp]: Do we include this for historical reasons or is this inaccurate?

**Comment by avarsava** [comment]
I've checked out this branch to test out the page but when I `mvn install`, 

HibernateSequencingContainerModelDaoTest.testGetUsagePlatform:151 expected:<4> but was:<0>

File: miso-web/src/main/webapp/WEB-INF/pages/editInstrumentModel.jsp (hunk 1 above)
Comment (djcooke) [reply miso-web/src/main/webapp/WEB-INF/pages/editInstrumentModel.jsp]: Was there because copy+paste. Removed

**Comment by avarsava** [comment]
I cannot set 'Sequencing Containers Per Run' to anything other than 0 - on both new and existing models, it says 'Should be 1 or greater for sequencer models'. The instrument type is set to Sequencer. Not sure I understand the error.
Adding a Run Position with an alias of >10 characters causes 'Something has gone terribly wrong, please report this to your MISO administrator'. Also can't remove the offending alias without refreshing the page and losing all my work - 'Nothing selected'.

**Comment by avarsava** [comment]
Does being able to set the 'Index Sequencing' value make sense for Array Scanner and Other instrument types?

**Comment by djcooke** [comment]
@avarsava: all fixed

**Comment by avarsava** [comment]
I tried editing a Sequencer to use 128 Sequencing Containers per Run and this causes 'Something has gone terribly wrong'. 127 is fine?

**Comment by djcooke** [comment]
@avarsava fixed this case, though similar things will probably happen when overflowing any datatype since we've never had validation to prevent it.

Also found and fixed an SQLException that was happening when lowering the number of containers if there were no instruments of the model being edited.

**Pull request merged**